charts/values.yaml

```diff
@@ -120,6 +124,10 @@ controllers:
     workers: 3
   secret:
     workers: 10
+  etcdOpsTask:
+    enabled: false
```

Comment:
Why set it to `false` here? We can have consistency across operatorConfig based and CLI based setups.